pycytominer/cyto_utils/features.py

```diff
@@ -66,19 +66,17 @@ def label_compartment(cp_features, compartment, metadata_cols):
     return cp_features
 
 
-def infer_cp_features(population_df, metadata=False):
+def infer_cp_features(
+    population_df, compartments=["Cells", "Nuclei", "Cytoplasm"], metadata=False
+):
     """
     Given a dataframe, output features that we expect to be cell painting features
     """
-    features = [
-        x
-        for x in population_df.columns.tolist()
-        if (
-            x.startswith("Cells_")
-            | x.startswith("Nuclei_")
-            | x.startswith("Cytoplasm_")
-        )
-    ]
+    features = []
+    for col in population_df.columns.tolist():
+        for ind, val in enumerate(compartments):
+            if col.startswith(compartments[ind].title()):
+                features.append(col)
```

Comment:
Looks great @hillsbury - I've proposed a small change below to make the code more [pythonic](https://docs.python-guide.org/writing/style/).

```suggestion
        if any([col.startswith(x.title()) for x in compartments]):
            features.append(col)
```

I didn't test this, but I think it should work